tctl.yaml

```diff
@@ -38,10 +38,26 @@ pipeline:
 
       make build
       install -Dm755 tctl ${{targets.destdir}}/usr/bin/tctl
-      install -Dm755 tctl-authorization-plugin ${{targets.destdir}}/usr/bin/tctl-authorization-plugin
```

Comment:
contextdir is a kind of special variable and we use that in melange pipelines because you can't know where the pipeline is going to be run so if it is used in the main package pipelines it would be `targets.destdir` if it is sub package it would be `targets.subpkgdir`, what I'm trying to say is that we mostly avoid using targets.contexdir direclty in the packages. but PTAL @mamccorm

Reply:
@rawlingsj  requested this change here https://github.com/wolfi-dev/os/pull/8897#discussion_r1407972076

Reply:
`contextdir` is expected to be used instead of `targets.destdir` and `targets.subpkgdir` - unless there is a specific reason that a pipeline needs to write to another packages output directory.

There's a bit more info https://github.com/chainguard-dev/melange/pull/622